### PR TITLE
Development friction fixes

### PR DESCRIPTION
**Pull request by TrevorDev**
https://github.com/BabylonJS/Babylon.js/issues/4267

### CODE REVIEW

File: src/Loading/babylon.sceneLoader.ts [hunk 1]
```diff
@@ -378,16 +378,26 @@
         /**
          * Import meshes into a scene
          * @param meshNames an array of mesh names, a single mesh name, or empty string for all meshes that filter what meshes are imported 
-         * @param rootUrl a string that defines the root url for scene and resources
-         * @param sceneFilename a string that defines the name of the scene file. can start with "data:" following by the stringified version of the scene
+         * @param rootUrl a string that defines the root url for scene and resources OR the concatination of rootURL and filename (eg. http://example.com/test.glb)
```
Comment (bghgary) [inline src/Loading/babylon.sceneLoader.ts]: spelling: concatenation

File: src/Loading/babylon.sceneLoader.ts [hunk 2]
```diff
@@ -378,16 +378,26 @@
...
          */
-        public static ImportMesh(meshNames: any, rootUrl: string, sceneFilename: string, scene: Scene, onSuccess: Nullable<(meshes: AbstractMesh[], particleSystems: ParticleSystem[], skeletons: Skeleton[], animationGroups: AnimationGroup[]) => void> = null, onProgress: Nullable<(event: SceneLoaderProgressEvent) => void> = null, onError: Nullable<(scene: Scene, message: string, exception?: any) => void> = null, pluginExtension: Nullable<string> = null): Nullable<ISceneLoaderPlugin | ISceneLoaderPluginAsync> {
+        public static ImportMesh(meshNames: any, rootUrl: string, sceneFilename: any = "", scene: Nullable<Scene> = Engine.LastCreatedScene, onSuccess: Nullable<(meshes: AbstractMesh[], particleSystems: ParticleSystem[], skeletons: Skeleton[], animationGroups: AnimationGroup[]) => void> = null, onProgress: Nullable<(event: SceneLoaderProgressEvent) => void> = null, onError: Nullable<(scene: Scene, message: string, exception?: any) => void> = null, pluginExtension: Nullable<string> = null): Nullable<ISceneLoaderPlugin | ISceneLoaderPluginAsync> {
+            if(!scene){
+                Tools.Error("No scene availible to LoadAssetContainer to");
```
Comment (bghgary) [inline src/Loading/babylon.sceneLoader.ts]: copy & paste error: LoadAssetContainer -> ImportMesh
spelling: available

File: src/Loading/babylon.sceneLoader.ts [hunk 3]
```diff
@@ -378,16 +378,26 @@
...
+                return null;
+            }
+
+            if(sceneFilename == ""){
```
Comment (bghgary) [inline src/Loading/babylon.sceneLoader.ts]: Why not use null instead of empty string?

File: src/Loading/babylon.sceneLoader.ts [hunk 4]
```diff
@@ -476,14 +486,14 @@
         /**
         * Import meshes into a scene
         * @param meshNames an array of mesh names, a single mesh name, or empty string for all meshes that filter what meshes are imported 
-        * @param rootUrl a string that defines the root url for scene and resources
-        * @param sceneFilename a string that defines the name of the scene file. can start with "data:" following by the stringified version of the scene
+        * @param rootUrl a string that defines the root url for scene and resources OR the concatination of rootURL and filename (eg. http://example.com/test.glb)
```
Comment (deltakosh) [inline src/Loading/babylon.sceneLoader.ts]: same typo

**Pull request merged**